### ISSUE TITLE
Issue using last day of month `L` symbol and queueing using a CRON.

### ISSUE THREAD

**Issue by matthieuh** [quirrel-dev/quirrel#230]
## Bug Report

**Current Behavior**
Queueing using a CRON using last day of month `L` symbol (`0 0 10 L * *`) it seems that this is taking the last day of the current month. Example queuing it today (03/31/2021)  with `0 0 10 L * *` In the UI I see the following CRON scheduled: `0 0 10 30 * *`

**Input Code**

- REPL or Repo link if applicable:

```ts
queue.enqueue(
  {  ...  },
  {
    id: `xxx`,
    override: true,
    repeat: {
      cron: `0 0 10 L * *`
    }
  }
)
```

**Expected behavior/code**
I would expect to have a job executed last day of each month.

**Environment**

- Quirrel version: v1.3.3
- Node/npm version: v12.13.1

**Comment by Skn0tt** [comment]
Hi @matthieuh! I've added written two small acceptance tests (https://github.com/quirrel-dev/quirrel/pull/232), and it seems like the problem is not on `cron-parser`'s side.

I also can't replicate the issue on my end. Using your provided snippet, the cron expression is enqueued correctly:
<img width="766" alt="Screenshot 2021-03-31 at 08 59 03" src="https://user-images.githubusercontent.com/14912729/113103290-627d5100-91ff-11eb-96aa-a2d619c62a32.png">

Could you provide me with a repro? What I can't reproduce, I can't fix 😬

**Comment by matthieuh** [comment]
Thank you Simon for having tried to reproduce it and unfortunately (or not) I also can't reproduce it anymore 😅  So probably an issue on my side...

Sorry for your time...

**Comment by Skn0tt** [comment]
I think it could have to do with a display issue in the UI. I'll look into that :)

**Comment by Skn0tt** [comment]
I've been able to recreate the issue and fixed it via #230.
